### PR TITLE
adjust discount of altarapay product

### PR DESCRIPTION
**Pull request by darmhoo**
Adjust discount on altaraPay

### CODE REVIEW

File: src/components/PaymentLog.vue [hunk 1]
```diff
@@ -698,6 +703,16 @@ export default {
   },
   data() {
     return {
+      dummyDiscount: {
```
Comment (teebarg) [inline src/components/PaymentLog.vue:706]: Do we want to leave this here?

**Comment by teebarg** [comment]
So much formatting changes here, can you point to where your changes are?

File: src/components/PaymentLog.vue (hunk 1 above)
Comment (darmhoo) [reply src/components/PaymentLog.vue:706]: so that it can easily be changed if 7% is not appropriate

Comment (teebarg) [reply src/components/PaymentLog.vue:706]: is it coming from backend? the discount

**Review by teebarg — APPROVED** [review]

**Pull request merged**